### PR TITLE
Store bg summary 2

### PR DESCRIPTION
**Pull request by cdebourcy**
I re-did store-bg-summary due to the rebase issues and . Think this one should be good to go

### CODE REVIEW

File: app/controllers/backgrounds_controller.rb [hunk 1]
```diff
@@ -25,7 +25,10 @@ def edit
   # POST /backgrounds
   # POST /backgrounds.json
   def create
-    @background = Background.new(background_params)
+    @background = Background.create(background_params)
```
Comment (yunfangjuan) [inline app/controllers/backgrounds_controller.rb]: shouldn't this be inside the transaction if we want the whole thing to rollback if the creation failed?

Comment (cdebourcy) [reply app/controllers/backgrounds_controller.rb]: absolutely, thanks

File: app/controllers/backgrounds_controller.rb [hunk 2]
```diff
@@ -25,7 +25,10 @@ def edit
   # POST /backgrounds
   # POST /backgrounds.json
   def create
-    @background = Background.new(background_params)
+    ActiveRecord::Base.transaction do
```
Comment (yunfangjuan) [inline app/controllers/backgrounds_controller.rb]: Great!!

File: app/controllers/backgrounds_controller.rb [hunk 3]
```diff
@@ -69,6 +76,22 @@ def set_background
     @background = Background.find(params[:id])
   end
 
+  def store_summary
```
Comment (yunfangjuan) [inline app/controllers/backgrounds_controller.rb]: I think this function should really go into model/background.rb

**Comment by cdebourcy** [comment]
After merging and closing this PR, go to alpha.idseq.net/backgrounds/{id}/edit for each existing background id and click "Update" to create the relevant taxon_summaries.

**Pull request merged**